avocado/core/remote/runner.py

```diff
@@ -132,6 +132,45 @@ def check_remote_avocado(self):
         except IndexError:
             return (False, None)
 
+    @staticmethod
+    def _parse_json_response(output):
+        """
+        Try to parse JSON response from the remote output.
+
+        It tries to find start of the json dictionary and then grabs
+        everything till the end of the dictionary. It supports single-
+        line as well as multi-line pretty json output.
+        """
+        _result = iter(output.splitlines())
+        json_result = ""
+        response = None
+        for line in _result:    # Find the beginning
+            if line.startswith('{'):
+                json_result += line
+                break
+        else:
```

Comment:
you could not find the beginning of the json output because there's no json output... the message could be better, but I'm ok with that.